### PR TITLE
Update Knative Serving/Eventing stack to v1.1.0

### PR DESCRIPTION
**Pull request by lamw**
Closes: #509 

## Pull Request Checklist
🚨 Please review the [guidelines for contributing](https://vmweventbroker.io/community) to this repository.

- [ ] Please ensure that you are making a pull request against the **Development** branch
- [ ] Please use the `WIP` keyword in the title of your PR if you are not ready for review
- [ ] Please ensure that you have opened a Github Issue if you are resolving/fixing a problem
- [ ] Please ensure that you have [signed](https://help.github.com/en/github/authenticating-to-github/signing-commits) all commits and that you have [squashed](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec) all relevant commits related to your change
- [ ] Please make sure that you have tested your change locally by successfully [building and deploying the VMware Event Broker Appliance](https://vmweventbroker.io/kb/contribute-appliance) and/or [building and deploying VMware Event Router](https://vmweventbroker.io/kb/contribute-eventrouter)
- [ ] Please include any relevant screenshots and/or output as part of your testing
- [ ] Please include any documentation updates that is applicable for your changes

## Change Type

What types of changes does your code introduce to the VMware Event Broker Appliance?

_Put an `x` in all boxes that apply_

Please check the type of change your PR introduces:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation changes
- [ ] Other (please describe):

### CODE REVIEW

File: veba-bom.json [hunk 1]
```diff
@@ -87,47 +76,71 @@
...
             },
             {
                 "name": "gcr.io/knative-releases/knative.dev/serving/cmd/webhook@sha256",
-                "version": "07894526a4b49c61c5333e4f48190e830ce5124f0c18e1042cc15fcb594bfc37"
+                "version": "382a1d64ea0686da2b973c95be96fcae29ac25b256b90f5735b1479a93d19c7a"
+            }
+        ]
+    },
+    "knative-eventing": {
+        "gitRepoTag": "v0.25.1",
```
Comment (embano1) [inline veba-bom.json:112]: nit: align this with serving?

File: veba-bom.json [hunk 2]
```diff
@@ -87,47 +76,71 @@
...
             {
-                "name": "gcr.io/knative-releases/webhook-f31b5fdf3b3cdeb5a25aa0a5d79e4348@sha256",
-                "version": "eb480afef446b9f0b016cf82d88553c9a0adb4dde39505ca3d2e5830e79399e4"
+                "name": "gcr.io/knative-releases/knative.dev/eventing/cmd/webhook@sha256",
+                "version": "337f9964dbe870b822f994a95886a9dc6dcb075e8141e7efad7b3314990e1b71"
+            }
+        ]
+    },
+    "knative-contour": {
+        "gitRepoTag": "v0.25.1",
```
Comment (embano1) [inline veba-bom.json:132]: same w/ eventing

**Review by embano1 — CHANGES_REQUESTED** [review]
Can you please align all kn related component versions?

File: veba-bom.json (hunk 1 above)
Comment (lamw) [reply veba-bom.json:112]: IIRC, there wasn't v0.26 and latest (at the time of this PR) was v0.25.1

**Comment by lamw** [comment]
> Can you please align all kn related component versions?

Yup. Will align with v1.1.0

**Review by embano1 — APPROVED** [review]
LGTM 🚀

**Comment by lamw** [comment]
Squashed all commits

**Review by embano1 — APPROVED** [review]

**Pull request merged**